core/src/test/resources/WEB-INF/config-lucene.xml

```diff
@@ -125,6 +125,7 @@
       <field name="_valid_schematron-rules-iso" tagName="valid_schematron-rules-iso"/>
       <field name="_valid_schematron-rules-inspire" tagName="valid_schematron-rules-inspire"/>
       <field name="_valid_xsd" tagName="valid_xsd"/>
+      <field name="_valid_subtemplate" tagName="valid_subtemplate"/>
```

Comment:
We don't need this ? _valid contains the information about the validation status even for subtemplate ?